pkg/volume/awsebs/aws_ebs_block_test.go

```diff
@@ -78,8 +78,8 @@ func TestGetVolumeSpecFromGlobalMapPath(t *testing.T) {
 	}
 	block := v1.PersistentVolumeBlock
 	specMode := spec.PersistentVolume.Spec.VolumeMode
-	if &specMode == nil {
-		t.Errorf("Invalid volumeMode from GlobalMapPath spec: %v - %v", &specMode, block)
+	if specMode == nil {
+		t.Fatalf("Failed to get spec from GlobalMapPath: %v", err)
```

Comment:
Why change this to `Fatalf` and change the wording?

Reply:
@gnufied thanks for the review. Since
1. specMode is nil (there' no point proceeding further)
2. specMode is not able to located hence the message should reflect.

Thoughts ?

Reply:
It makes no difference in this case because using `Errorf` will also fail the test and stop running further statements.

Reply:
@gnufied , 
From the go doc:
Errorf is equivalent to Logf followed by Fail. Fail marks the function as having failed but continues execution. 
Fatalf is equivalent to Logf followed by FailNow. FailNow marks the function as having failed and stops its execution by calling runtime.Goexit .

changing it back to Errorf()